tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1628,6 +1628,15 @@ test_pktgen.py:
   skip:
     reason: "have known issue, skip for now"
 
+#######################################
+#####        upgrade_path         #####
+#######################################
+upgrade_path/test_upgrade_path.py::test_warm_upgrade_sad_path:
+  xfail:
+    conditions:
+      - https://github.com/sonic-net/sonic-mgmt/issues/13627
+      - "neighbor_type!='eos'"
```

Comment:
Should this be `eos` or `ceos`, or both?

Reply:
The only valid `neighbor_types` are `sonic`, `eos`, or `cisco`, so just `eos` is needed here.